server/src/test-data.js

```diff
@@ -1,5 +1,10 @@
 import Creators from './creators';
 
+const nowInGMT = () => {
+  const d = new Date();
+  return Math.round((d.getTime() - 39600) / 1000);
```

Comment:
you sure this is necessary? I thought that getTime() returned UTC timestamps (unix timestamps should always be in UTC)
either way this is fine for tests but we need to be careful with this stuff in the actual code.

Reply:
Researched... yes you are correct.